extensions/amp-ventuno-player/validator-amp-ventuno-player.protoascii

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-ventuno-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-ventuno-player"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-ventuno-player>
+  html_format: AMP
+  tag_name: "AMP-VENTUNO-PLAYER"
+  requires_extension: "amp-ventuno-player"
+  attrs: {
+    name: "data-player"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-pubid"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-slotid"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-title"
+    mandatory: false
```

Comment:
`data-` attributes are allowed by default, so unless it's mandatory or has a value constraint, then it doesn't need to be listed here. this goes for `data-title`, `data-url` and `data-meta`.